### ISSUE TITLE
Unable to create shapes in Word (with the mouse) while NVDA running.

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#6141]
Dear people of Earth! (specifically @michaelDCurran)

In Word (2016), From the Insert ribbon (alt+n) choose shapes (s then h), then choose a shape and press ENTER.  Normally the mouse pointer changes to the "precision" cursor.  You then click somewhere on your document and drag to size the shape you have selected (there isn't currently a keyboard way of completing this).

With NVDA running, the precision cursor reverts to a regular cursor as soon as NVDA tries to speak - essentially as soon as the focus returns to the document after selecting a shape to insert.

**Comment by Adriani90** [comment]
@Qchristensen are you still able to reproduce this issue in NVDA 2019.1 Beta 1?

**Comment by Qchristensen** [comment]
In Office 365 version 16.0.11601.20174 and NVDA 2019.1.1, the behaviour has changed:
- If you click on the "Shapes" item in the insert ribbon and then the one you want, the cursor changes to a precision cursor, and you can use the mouse to size and insert the shape wherever you want. (Therefore this specific issue is now resolved).
- If you use the keyboard to select a shape: ALT+n, then s, then h, then use the arrows to select a shape and ENTER to insert it, the shape is instantly inserted into the document at the current cursor position.

With the shape inserted via the keyboard, it is selected and you can use:
- ARROWS to move it (by about a millimetre).
- SHIFT+ARROWS to resize it (by a couple of millimetres)
- ALT+LEFT and RIGHT ARROWS to rotate it by 15 degrees (1/24th of a full rotation).

With the item selected, you can also access the shape format ribbon with alt, j, d to adjust other properties such as colour and bring to front.

None of that beyond selecting the shape in the first place (and accessing the ribbon items) is accessible - NVDA provides no feedback at all, so issue #5738 is still unresolved.

**Comment by Qchristensen** [comment]
Closing as WorksForMe (as per my comment last year on this, and my testing just now in Office 365 (64-bit) Version: 16.0.12827.20258  with NVDA 2020.1 confirms it still is).
